pkg/appregistry/adapter.go

```diff
@@ -0,0 +1,71 @@
+package appregistry
+
+import (
+	"bytes"
+
+	appr "github.com/operator-framework/go-appr/appregistry"
+	appr_blobs "github.com/operator-framework/go-appr/appregistry/blobs"
+	appr_package "github.com/operator-framework/go-appr/appregistry/package_appr"
+	appr_models "github.com/operator-framework/go-appr/models"
+)
+
+const (
+	mediaType string = "helm"
+)
+
+// This interface (internal to this package) encapsulates nitty gritty details of go-appr client bindings
```

Comment:
Comment could have been a bit more clear on why 3 methods.

Reply:
That's all we need for this iteration. Do you want more clarity on why this interface exists? I don't think we need to explain why an interface has N number of methods.